release_build_files/Android/firebase_dependencies.gradle

```diff
@@ -16,30 +16,30 @@ import org.gradle.util.ConfigureUtil;
 
 // A map of library to the dependencies that need to be added for it.
 def firebaseDependenciesMap = [
-  'app' : ['com.google.firebase:firebase-analytics:18.0.3'],
+  'app' : ['com.google.firebase:firebase-analytics:19.0.0'],
   'admob' : ['com.google.firebase:firebase-ads:19.8.0',
              'com.google.android.gms:play-services-measurement-sdk-api:18.0.3',
```

Comment:
The other one got changed to analytics:19.0.0, I assume this one needs to do the same

Reply:
Yep. Good catch!